torchtune/utils/_checkpointing/_checkpointer.py

```diff
@@ -401,6 +401,7 @@ def load_checkpoint(self) -> Dict[str, Any]:
                 num_heads=self._config["num_attention_heads"],
                 num_kv_heads=self._config["num_key_value_heads"],
                 dim=self._config["hidden_size"],
+                head_dim=self._config["head_dim"],
```

Comment:
@ebsmothers this is a "generic" silent error where head_dim was never given and always inferred to be `head_dim = dim // num_heads`

Reply:
You are right I think mistral does not have it in config : https://huggingface.co/mistralai/Mistral-7B-v0.1/blob/main/config.json